### PR TITLE
Volue/Gemini Terrain Sprint 2.1

### PR DESCRIPTION
**Pull request by wojciech-k-volue**
## File

File covering concepts from sprint 2.1. Includes screenshot showing the original alignment geometry.

TestTunnel_sprint_2 1

### CODE REVIEW

**Comment by pjanck** [comment]
Hi @wojciech-k-volue , try this: the `IfcReferent` shall be nested using its own `IfcRelNests` and not reuse the one that nests `IfcAlignmentHorizontal` and other parts.

**Comment by wojciech-k-volue** [comment]
@pjanck I followed your suggestion in regard to having a separate `IfcRelNests` for the `IfcReferents`, however it didn't seem to have an effect on the result.

**Comment by pjanck** [comment]
> it didn't seem to have an effect on the result.

I had a closer look at your example and tried it offline. There is a mistake in the checker! I will need to update the checker. When this happens, I'll restart the check and I believe your example should be green!

**Comment by wojciech-k-volue** [comment]
> > it didn't seem to have an effect on the result.
> 
> I had a closer look at your example and tried it offline. There is a mistake in the checker! I will need to update the checker. When this happens, I'll restart the check and I believe your example should be green!

Great, thanks for the help!

File: files/Volue_GeminiTerrain/TestTunnel_sprint_2.1.ifc [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+#231=IFCCARTESIANPOINT((298548.471716,6695621.794949));
+#232=IFCCARTESIANPOINT((298639.524412,6695674.278333));
+#233=IFCCARTESIANPOINT((298661.595193,6695716.439264));
+#234=IFCCARTESIANPOINT((298664.356956,6695722.653195));
+#235=IFCCARTESIANPOINT((298667.823706,6695730.497121));
+#236=IFCCARTESIANPOINT((298671.225284,6695738.369528));
+#237=IFCCARTESIANPOINT((298700.054361,6695805.471582));
+#238=IFCCARTESIANPOINT((298721.598948,6695847.903813));
+#239=IFCCARTESIANPOINT((298853.522683,6695898.569213));
+#240=IFCCARTESIANPOINT((298897.931702,6695881.466509));
```
[the remark this answers is not in the document]
Comment (wojciech-k-volue) [reply files/Volue_GeminiTerrain/TestTunnel_sprint_2.1.ifc]: For this sprint at least, I've set the offset in `IfcMapConversion` to (0, 0, 0). I'll consider properly reducing the coordinates for next sprints.

**Review by pjanck — APPROVED** [review]
Thank you for the sketch as well!

**Pull request merged**